src/chunkserver/raftlog/curve_segment.cpp

```diff
@@ -217,6 +217,8 @@ int CurveSegment::load(braft::ConfigurationManager* configuration_manager) {
 
     if (_is_open) {
         _last_index = actual_last_index;
+    } else {
+        _offset_and_term.shrink_to_fit();
```

Comment:
why add this?

Reply:
I done memory profiling before, the result shows `_offset_and_term` occupies lots of memory. And `shrink_to_fit` only reallocate necessary memory which makes `capacity() == size()`.